### DISCUSSION TITLE
CPU 80-90% loaded

### DISCUSSION THREAD

**Discussion by fadmaks** [pH7Software/pH7-Social-Dating-CMS#1166] [Troubleshooting]
I run ph7cms on a local server and after a short time the CPU becomes 80-90% loaded. A similar situation occurs when connecting to a third-party server. There is a heavy load on the user's processor. In doing so, I used different browsers.
CPU load is also observed during program installation.
Where to look for the error?

**Comment by SosthenesAlpha** [comment]
Disable background video.
It solved it for me somehow 🤔🤔
I ONLY have issues with registation process and I am yet to get a solution to it.
